qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1185,6 +1193,8 @@ def update(self, md_template):
                     "There are no differences between the data stored in the "
                     "DB and the new data provided",
                     QiitaDBWarning)
+                return
```

Comment:
Please correct me if I'm wrong, but it looks like the actual check `npt.assert_warns` is deleted? `test_update_fewer_samples`